### ISSUE TITLE
Major issue and recommendation

### ISSUE THREAD

**Issue by SoulThread2597** [KhamisiKibet/QT-PyQt-PySide-Custom-Widgets#25]
I've seen (and also experienced) multiple issues with this library, Here are some of them:

1. Sys not imported in some files, Fix: import sys in the files that need it
2. 3.10 Mapping with collections, Fix: not much you can do but tell people to change the collections to collections.abc in the file it is complaining about
3. Svg2Png conversion, this one is a huge issue, beacuse yes, gtk is needed for CairoSVG, but, for people like me, i dont want people to have to download another system ontop of all the ones for my project, and so my work around was to completely disabled that script from being used, which means i cant use anything related to themes now, Fix: possable switch to an easier and more reliable Svg converter, maybe even something like [svglib](https://pypi.org/project/svglib/)

These are all the issue and possible fixes that do need resolved

**Comment by KhamisiKibet** [comment]
Any recommendations on the SVG converter?

**Comment by 3bears-data** [comment]
I'm facing problems with cairo.lib cause I'm not getting to install the lib for them. How can I fix it?

Py 3.7

**Comment by 3bears-data** [comment]
> I'm facing problems with cairo.lib cause I'm not getting to install the lib for them. How can I fix it?
> 
> Py 3.7 Windows 10 64bits

Well, I just entered the script where cairo is imported and commented the import and the functions being called by it. I still don't know the impact, but it did.

**Comment by islamshahine** [comment]
I have the same problem, can you explain more how to fix the follwoing:
no library called "cairo-2" was found
no library called "cairo" was found
no library called "libcairo-2" was found
cannot load library 'libcairo.so.2': error 0x7e
cannot load library 'libcairo.2.dylib': error 0x7e
cannot load library 'libcairo-2.dll': error 0x7e

**Comment by 3bears-data** [comment]
I had to find where script cairosvg was installed into my o.s. As I'm working with VisualCode IDE, it reveals the path location to SvgToPngIcons.py - I guess this script is used to convert the icons from png to SVG. 

In my environment, that script was installed in this folder: c:\users\{username}\appdata\local\programs\python\python37\lib\site-packages\custom_widgets\qss\svgtopngicons.py

So, with the script opened, I commented the first line where was importing cairosvg and every line with "cairosvg." {dot after cairosvg}, I commented too and write another line below "cairosvg." with the command None.

I know, it's not the best solution but It's worked for me. //file with the changes attached


[SvgToPngIcons.zip](https://github.com/KhamisiKibet/QT-PyQt-PySide-Custom-Widgets/files/10385583/SvgToPngIcons.zip)
